Template10 (Library)/Services/NavigationService/NavigationService.cs

```diff
@@ -290,8 +299,16 @@ public void Navigate<T>(T key, object parameter = null, NavigationTransitionInfo
             state.Write<string>("NavigateState", FrameFacadeInternal?.NavigationService.NavigationState);
         }
 
-        public event TypedEventHandler<Type> AfterRestoreSavedNavigation;
-        public async Task<bool> RestoreSavedNavigationAsync()
+        public event TypedEventHandler<Type> OnNavigationStateLoaded;
```

Comment:
Okay, so I see you are wanting to make the name of the event better. I agree. This could potentially impact other things. Effectively changing the signature of the type without a killer reason. If you don't have a compelling reason to change it, please leave it the same. 

As an aside, the MSDN naming guidelines now suggest omitting the "on" prefix, so a better name would be `NavigationStateLoaded` which implies after and `NavigationStateLoading` which implies a cancelled before event.